docs/03-design/LZ-03-08.adoc

```diff
@@ -1,105 +1,62 @@
-
-
-Softwarearchitekt:innen kennen wesentliche Quellen für Architekturmuster, beispielsweise die POSA-Literatur (z.{nbsp}B. <<buschmanna>>) und PoEAA (<<fowler>>) (für Informationssysteme) (R3)
+* {glossary_url}blackboard[Blackboard]
+* {glossary_url}broker[Broker]
+* {glossary_url}cqrs[CQRS (Command-Query-Responsibility-Segregation)]
+* {glossary_url}event-sourcing[Event sourcing]
+* Integrations und Messaging-Patterns (z.B. aus <<hohpe>>)
+* {glossary_url}model-view-controller[MVC] (Model View Controller),
+  {glossary_url}model-view-viewmodel[MVVM] (Model View ViewModel),
+  {glossary_url}model-view-update[MVU] (Model View Update),
+  {glossary_url}presentation-abstraction-control[PAC] (Presentation Abstraction Control)
+* {glossary_url}plugin[Plugin]
+* {glossary_url}ports-and-adapters[Ports and Adapters] (Synonyme: Onion Architektur, Hexagonal Architektur, Clean Architektur)
+* {glossary_url}service-oriented-architecture[SOA] (Service-Oriented Architektur)
+
+Softwarearchitekt:innen kennen wesentliche Quellen für Architekturmuster, beispielsweise die POSA-Literatur (z.{nbsp}B. <<buschmanna>>) und PoEAA (<<fowler>>) (für Informationssysteme). (R3)
 
 // end::DE[]
 
 // tag::EN[]
 [[LG-03-08]]
-==== LG 03-08 [previously LG 2-05]: Describe, Explain and Appropriately Apply Important Patterns (R1, R3)
-
-Software architects know:
-
-* various architectural patterns and can apply them when appropriate
-* that patterns are a way to achieve certain qualities for given problems and requirements within given contexts
-* that various categories of patterns exist (R3)
-* additional sources for patterns related to their specific technical or application domain (R3)
-
-
-Software architects can explain and provide examples for the following patterns (R1):
 
-* _layers_:
-** abstraction layers hide details, example: ISO/OSI network layers, or "hardware abstraction layer". See https://en.wikipedia.org/wiki/Hardware_abstraction
-** another interpretation are Layers to (physically) separate functionality or responsibility, see https://en.wikipedia.org/wiki/Multitier_architecture
+==== LG 03-08 [previously LG 2-05]: Describe, Explain and Apply Important Architectural Patterns (R1, R3)
 
-* _pipes and filters_: representative for data flow patterns, breaking down stepwise processing into a series of processing-activities ("Filters") and associated data transport/buffering capabilities ("Pipes").
-* _microservices_ split applications into separate executables that communicate via a network
-* _dependency injection_ as a possible solution for the Dependency-Inversion-Principle <<newman>>
+Software architects can explain and provide examples for the following architectural patterns (R1):
 
+* {glossary_url}layer[Layer]
+* {glossary_url}pipes-and-filters[Pipes and Filters]
+* {glossary_url}microservice[Microservice]
+* {glossary_url}dependency-injection[Dependency Injection]
```

Comment:
Is this reeally an "architectural pattern": The other patterns in this list all prescribe a certain organization structure, this one doesn't. I suggest just removing it here, since it is mentioned elsewhere as R1.

Reply:
I removed dependency injection. It's part of LG-03-06 (R1). The original issue #424 also stated that dependency injection should be moved.